.github/workflows/apply-migrations.yaml

```diff
@@ -0,0 +1,32 @@
+name: Apply Database Migrations
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - 'db/schema.ts'
+
+jobs:
+  check-migrations:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          
+      - name: Install dependencies
+        run: npm install
+        
+      - name: Generate local migrations
+        run: npm run db:generate
+        env:
+          POSTGRES_URL: ${{ secrets.POSTGRES_URL }}
+          
+      - name: Preview database migrations
+        run: npm run preview-migrations
```

Comment:
Actually - call this `run-migrations`. The fact that it's a preview is only temporary.

Actually, change it to `npm run run-migrations -- --preview`, such that it currently previews with the `--preview` flag and will run without it